docs/aurora/data-management/daos/daos-overview.md

```diff
@@ -45,154 +66,303 @@ Total size: 6.0 TB
-More examples are available in `/soft/daos/examples`.
-The following example uses two support scripts, `launch-dfuse.sh` and `clean-dfuse.sh`, to startup dfuse on each compute node and then shut it down at job end, respectively.
 
-Job Script Example:
 ```bash
-#!/bin/bash
-#PBS -A <project>
-#PBS -lselect=1
-#PBS -lwalltime=30:00
-#PBS -k doe
-#
-# Test case for MPI-IO code example
+launch-dfuse.sh ${DAOS_POOL_NAME}:${DAOS_CONT_NAME} # launched using pdsh on all compute nodes mounted at: /tmp/<pool>/<container>
+mount | grep dfuse # To confirm if its mounted
 
-# ranks per node
-rpn=4
+ls /tmp/${DAOS_POOL}/${DAOS_CONT}/
 
-# threads per rank
-threads=1
+clean-dfuse.sh  ${DAOS_POOL_NAME}:${DAOS_CONT_NAME} # To unmount on all nodes 
+```
+DAOS Data mover instruction is provided at [here](../moving_data_to_aurora/daos_datamover.md).
 
-# nodes per job
-nnodes=$(cat $PBS_NODEFILE | wc -l)
+## Job Submission
 
-# Verify the pool and container are set
-if [ -z "$DAOS_POOL" ];
-then
-    echo "You must set DAOS_POOL"
-    exit 1
-fi
+The `-ldaos=default` switch will ensure that DAOS is accessible on the compute nodes.
 
-if [ -z "$DAOS_CONT" ];
-then
-    echo "You must set DAOS_CONT"
-    exit 1
-fi
+Job submission without requesting DAOS:  
+```bash
+qsub -l select=1 -l walltime=01:00:00 -A Aurora_deployment -k doe -l filesystems=flare -q lustre_scaling ./pbs_script1.sh  or - I 
+```
 
-# load daos/base module (if not loaded)
-module load daos/base
+Job submission with DAOS: 
+```bash
+qsub -l select=1 -l walltime=01:00:00 -A Aurora_deployment -k doe -l filesystems=flare -q lustre_scaling -l daos=default 	./pbs_script1.sh  or - I 
+```
 
-# print your module list (useful for debugging)
-module list
 
-# print your environment (useful for debugging)
-#env
+## NIC and Core Binding
 
-# turn on output of what is executed
-set -x
+Each Aurora compute node has 8 NICs and each DAOS server node has 2 NICs. 
+Each NIC is capable of driving 20-25 GB/s unidirection for data transfer. 
+Every read and write goes over the NIC and hence NIC binding is the key to achieve good performance. 
 
-#
-# clean previous mounts (just in case)
-#
-clean-dfuse.sh ${DAOS_POOL}:${DAOS_CONT}
+For 12 PPN, the following binding is recommended.
 
-# launch dfuse on all compute nodes
-# will be launched using pdsh
-# arguments:
-#   pool:container
-# may list multiple pool:container arguments
-# will be mounted at:
-#   /tmp/\<pool\>/\<container\>
-launch-dfuse.sh ${DAOS_POOL}:${DAOS_CONT}
+```bash
+CPU_BINDING1=list:4:9:14:19:20:25:56:61:66:71:74:79
+```
+![Sample NIC to Core binding](core-NIC-binding.png "Sample NIC to Core binding")
+
+
+
+## Interception library for posix containers
+
+The interception library (IL) is a next step in improving DAOS performance. This provides kernel-bypass for I/O data, leading to improved performance.
+The IL will intercept basic read and write POSIX calls while all metadata calls still go through dFuse.
```

Comment:
For libpil4dfs.so, meta data will also be intercepted?

Reply:
updated.